hypervisor/dm/vrtc.c

```diff
@@ -536,8 +576,38 @@ static inline bool vrtc_is_time_register(uint32_t offset)
 }
 
 /**
+ * @brief Write a value to the virtual RTC.
+ *
+ * This function writes a specified value to the virtual RTC at a given offset. To write to the virtual RTC, the guest
+ * writes the register index to the RTC address port and then writes the register value to the RTC data port. This
+ * function is used to simulate the behavior of writing in a virtualized environment.
+ *
+ * - If the accessed port is CMOS_ADDR_PORT and the width is 1 byte, it will store the value as the index and return.
+ * - If the accessed port is CMOS_DATA_PORT,
+ *   - For Service VM, it will directly write the value to the physical CMOS register. If the physical date/time is
+ *     changed, for RT VMs and pre-launched VMs, the RTC/TSC snapshots will be updated. Those snapshots are used to
+ *     emulate the virtual date/time for non-Service VM.
+ *   - For a non-Service VM, it will ignore the write to the RTC_STATUSA, RTC_INTR and RTC_STATUSD. Otherwise, it will
+ *     update the register value and rtc time. And for Post-launched VM, it will send a VM event to notify the VM of the
+ *     change in the RTC time if the address port is in the range of the time registers.
+ *
+ * @param[inout] vcpu Pointer to the virtual CPU that is writing to the virtual RTC.
+ * @param[in] addr The address port to write to.
+ * @param[in] width Width of the value to be written to the virtual RTC.
+ * @param[in] value Value to be written to the virtual RTC.
+ *
+ * @return A boolean value indicating whether the write operation is successful.
+ *
+ * @retval true Successfully write to the virtual RTC.
+ * @retval false Failed to write to the virtual RTC.
```

Comment:
In current implementation, it always returns `true`. Let's remove `@retval false ...` and revise the description a little bit.
Something like below:
```
@return A boolean value indicating whether the write operation is handled successfully, which is always true
        in current design. It either updates the physical registers, updates the virtual registers,
        or ignores the write.

@retval true The write operation is handled successfully.
```